src/managers/hub/SimplePriceManager.sol

```diff
@@ -88,21 +88,34 @@ contract SimplePriceManager is ISimplePriceManager {
 
         NetworkMetrics storage networkMetrics_ = networkMetrics[poolId][centrifugeId];
         Metrics storage metrics_ = metrics[poolId];
-        uint128 issuance = shareClassManager.issuance(poolId, scId, centrifugeId);
-
-        metrics_.issuance = metrics_.issuance + issuance - networkMetrics_.issuance;
-        metrics_.netAssetValue = metrics_.netAssetValue + netAssetValue - networkMetrics_.netAssetValue;
+        uint128 newIssuance = shareClassManager.issuance(poolId, scId, centrifugeId);
+
+        // When shares are transferred, the issuance in the SCM updates immediately,
+        // but in this contract they are tracked separately as transferredIn/Out.
+        // Here we get the diff between the old SPM issuance and the new SCM issuance,
```

Comment:
Nit
```suggestion
        // Here we get the diff between the old SCM issuance and the new SCM issuance,
```

Reply:
This is intentional (Single Price Manager = SPM)